css/styles.css

```diff
@@ -217,32 +219,59 @@ figure.chuck-container {
   padding-left: 25px;
   padding-bottom: 0;
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
 }
 
-.search-category {
+.search-category, .search-keyword {
   display: flex;
   justify-content: center;
   align-items: center;
+  gap: 10px;
+}
+
+#category {
+  line-height: 35px;
+  background-image:
+    linear-gradient(45deg, transparent 50%, gray 50%),
+    linear-gradient(135deg, gray 50%, transparent 50%),
+    linear-gradient(to right, #ccc, #ccc);
+  background-position:
+    calc(100% - 20px) calc(1em + 2px),
+    calc(100% - 15px) calc(1em + 2px),
+    calc(100% - 2.5em) 0.5em;
+  background-size:
+    5px 5px,
+    5px 5px,
+    1px 1.5em;
+  background-repeat: no-repeat;
+}
+
+select, input {
+  border: 1px solid rgba(0, 0, 0, 0.65);
```

Comment:
Borders can be left as 1px, not converted to rems just so you know.

Reply:
Sounds good! Thanks :)